mlbstatsapi/mlbapi.py

```diff
@@ -704,10 +704,12 @@ def get_object(self, object):
         # If problem with hydrating object, return the old dry object
         return hydratedobject if hydratedobject else object
 
-    def get_stats(self, object : Union[object, dict], params : dict) -> List[Stats]:
+    def get_stats(self, params : dict, mlb_object : Union[Union[Team, Person], dict] = NoneType) -> List['Splits']:
```

Comment:
Should this be `-> Dict{str, 'splits'}`?

Reply:
Or something along the lines of that

Reply:
hmmm yeah for sure I see

Reply:
one sec

Reply:
Should be all set now @KCNilssen